shakedown/dcos/__init__.py

```diff
@@ -30,12 +29,30 @@ def dcos_service_url(service):
     return _gen_url("/service/{}".format(service))
 
 
+def dcos_exhibitor_url():
+    return "http://{}/".format(master_ip())
```

Comment:
the issue is the function name.  I will use this function all the time for other service endpoints which have nothing to do with the exhibitor.

for instance MoM on dcos would be ` "{}/service/{}".format(dcos_exhibitor_url(), "marathon-user")`
it seems better to be: ` "{}/service/{}".format(dcos_admin_router_url(), "marathon-user")`

it would also be better to have 

```
def dcos_service_url():
 +    return "http://{}/service".format(master_ip())
```